src/olympia/reviews/views.py

```diff
@@ -65,12 +66,16 @@ def review_list(request, addon, review_id=None, user_id=None, template=None):
     else:
         ctx['page'] = 'list'
         qs = qs.filter(is_latest=True)
+        # Don't filter out empty reviews for admins.
+        if not is_admin:
+            # But otherwise, filter out everyone elses empty reviews.
+            qs = qs.filter(~Q(body=None) | Q(user=request.user.pk))
```

Comment:
There is no guarantee that `request.user` is not a `AnonymousUser` at this point is there ? We shouldn't have reviews with `user=None` but IMHO it's best to avoid doing that condition entirely for anonymous users